configmaptocrs/README.md

```diff
@@ -92,3 +93,52 @@ status: {}
 ---
 
 ```
+
+## Running directly against a cluster
+
+Configmaptocrs tool can also run directly against a cluster,
+and print the CRs corresponding to the configmap deployed on
+the cluster to the standard output.
+```
+kubectl run configmaptocrs -n metallb-system --restart=Never -it --rm --image overriden --overrides '
+{
+  "spec": {
+    "containers": [
+      {
+        "name": "configmaptocrs",
+        "image": "quay.io/metallb/configmaptocrs",
+        "command": [
+          "/configmaptocrs",
+          "-source",
```

Comment:
should we add config-as-yaml too here?

Reply:
This naming is confusing and not distinguishing IMO. both approaches use YAML,
we have the ConfigMap YAML or the ConfigMap's data YAML.